src/index.ts

```diff
@@ -154,6 +173,30 @@ const extension: JupyterFrontEndPlugin<void> = {
         // Check if the new value is an instance of NotebookPanel
         if (newValue instanceof NotebookPanel) {
           newValue.title.changed.connect(onTitleChanged);
+        } else if (
+          (newValue.title.label === 'Launcher' ||
+            newValue.title.label === 'Config Setup' ||
+            newValue.title.label === 'Clusters' ||
+            newValue.title.label === 'Serverless') &&
+          lastClusterName !== ''
+        ) {
+          localStorage.setItem('oldNotebookValue', lastClusterName || '');
```

Comment:
what is this code for?  Is this for updating the dpms widget when the notebook changes?
ifthat's the case we should really be passing the inotebooktracker to the dpmswidget and have it listen to it instead of listen to the events in the plugin, updating localstorage and have the dmps lsiten ot localstorage changes.

Reply:
This is to track previous notebook in case if we are going to launcher screen and coming back to any of the opened table/database details we need the last notebook value.